pkg/secrets/secrets.go

```diff
@@ -40,13 +40,11 @@ func (s *secertManager) GetSynapseName() string {
 }
 
 func (s *secertManager) WriteGitSecrets(path string) error {
-	gitSecrets := secretsFile{
```

Comment:
Shouldn't we also delete `secretsFile` struct?

Reply:
Apparently `secretsFile` is used in repo secret in by both synapse and nucleus

Reply:
Suggestion: Let remove secretFile from repo secrets too.

Reply:
removed `secretsFile`